src/app/edit-recipe/edit-recipe.component.ts

```diff
@@ -28,10 +28,10 @@ export class EditRecipeComponent implements OnInit {
   constructor(
     public firebaseService: FirebaseService,
     private route: ActivatedRoute,
-    private router: Router
-  ) {
+    private router: Router,
+  )
+  {
```

Comment:
`{` should be in the same line as `)` :) This is preferred in JS, as opposed to C, C++, C# :)